docs/components/clients/users.ts

```diff
@@ -469,4 +469,13 @@ export const users: Array<TurboUser> = [
     infoLink: "https://backpack.app",
     pinned: true,
   },
+  {
+    caption: "Supernova.io",
+    image: "/images/logos/supernova.svg",
+    infoLink: "https://supernova.io",
```

Comment:
We did a pass to make sure we don't 301 on the link, checked the same here.

```suggestion
    infoLink: "https://www.supernova.io/",
```